open-metadata-implementation/access-services/cognos/cognos-client/pom.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!-- SPDX-License-Identifier: Apache-2.0 -->
+<!-- Copyright Contributors to the ODPi Egeria project.  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
```

Comment:
As above, please add an SCM section - ditto on the naming

Reply:
done https://github.com/odpi/egeria/pull/3044/commits/2c7f52af9a67feb496eedd853b634a9b5f478b2f